homeassistant/components/lyric/strings.json

```diff
@@ -41,6 +41,17 @@
       },
       "setpoint_status": {
         "name": "Setpoint status"
+      },
+      "avg_temperature": {
+        "name": "Average temperature"
+      },
+      "avg_humidity": {
+        "name": "Average humidity"
+      }
+    },
+    "binary_sensor": {
+      "overall_motion": {
+        "name": "Overall motion"
```

Comment:
"Overall motion" read a bit funny, why not's just "motion"?

Reply:
You're right, it does read funny, and it might not translate very well either. 

I used the actual term that the Lyric API uses (the field is `overallMotion`). They use that term because the value is an aggregation of one or more motion sensors. If you had two sensors in one room, for example, this would be true if either sensor detected motion.

Similar reason why the above sensors are "Average temperature" and not just "Temperature". It's not accurate to say "average" for the motion, though. 

With that context, if it still sounds too weird I can just call it "Motion".